org.keshavnrj.wonderwall.yaml

```
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

If you do not support wayland then remove the wayland line and keep this.

---

If i user 5.15 where wayland is supported, should I keep wayland? Also What if I keep all three of these  
  ```
- --socket=x11
  - --socket=fallback-x11
  - --socket=wayland
```